README.md

```diff
@@ -36,7 +36,7 @@ When playing TGC, there are always cool "look what the AI did this time" moments
 
 Banks and Universities provide the player with new playstyles and methods to spend money. Universities are costly, but they provide prestige and a steady source of research points,  giving the player the opportunity of pulling ahead on research and becoming a technological powerhouse. Development banks will assist local landowners in developing their farms and considerably increasing farming capacity, allowing the player to invest more in RGO rather than just industrialisation.
 
-With Pioneers and Tribesmen the player will have an unique expansion and colonization experience. Pioneers, usually seen on the American frontier, are a special pop that can work as both a farmer and a soldier, a worker always ready to get his trusty rifle and protect his land. Tribesmen are similarly unique pops; they are difficult to integrate and have no plans in working for their colonizers, cause of course, why would a native pick up a hoe and start farming as soon as someone claim their lands?
+With Pioneers and Tribesmen the player will have an unique expansion and colonization experience. Pioneers, usually seen on the American frontier, are a special pop that can work as both a farmer and a soldier, a worker always ready to get his trusty rifle and protect his land. Tribesmen are similarly unique pops; they are difficult to integrate and have no plans in working for their colonizers, 'cause of course, why would a native pick up a hoe and start farming as soon as someone claims their lands?
```

Comment:
Suggestion:;
Our two new **PoPs**: Pioneers and Tribesmen allow for a never-seen experience when it comes to expanding and colonizing new lands. Pioneers, who are usually seen in lands ripe for colonization, [...]
Tribesmen are similarily unique **PoPs**, in the sense that they are harder to integrate into western society, and have no plans in working with settlers. Why would a native pick up a hoe and start farming as soon as someone steps into their lands?
These two new PoPs open room to vast possibilities in terms of flavor, immersion, and gameplay, something which you may never see in any other mod.